sql/postgresql/PostgreSQLParser.g4

```diff
@@ -2033,10 +2003,7 @@ opt_nulls_order
    ;
 
 createfunctionstmt
-   : CREATE opt_or_replace FUNCTION func_name func_args_with_defaults RETURNS func_return createfunc_opt_list
-   | CREATE opt_or_replace FUNCTION func_name func_args_with_defaults RETURNS TABLE OPEN_PAREN table_func_column_list CLOSE_PAREN createfunc_opt_list
-   | CREATE opt_or_replace FUNCTION func_name func_args_with_defaults createfunc_opt_list
-   | CREATE opt_or_replace PROCEDURE func_name func_args_with_defaults createfunc_opt_list
+   : CREATE opt_or_replace (FUNCTION | PROCEDURE) func_name func_args_with_defaults (RETURNS (func_return | TABLE OPEN_PAREN table_func_column_list CLOSE_PAREN))? createfunc_opt_list
```

Comment:
Please use breaks if the line length exceeds 120 chars. It simplifies grammar editing and review.

```suggestion
   : CREATE opt_or_replace (FUNCTION | PROCEDURE)
     func_name func_args_with_defaults
     (RETURNS (func_return | TABLE OPEN_PAREN table_func_column_list CLOSE_PAREN))? createfunc_opt_list
```

Reply:
i'm using antlr formatter from antlr repository, it formats grammars this way
I think there are 2 options:
1. make grammar stable (or at least without further major changes) and format it once manually
2. Change (somehow) formatter to make more fancy formatting, but I still wondering "which formatting is better"
3